src/Stripe.net/Services/Sources/StripeSourceCreateOptions.cs

```diff
@@ -44,6 +44,9 @@ public class StripeSourceCreateOptions : StripeBaseOptions, ISupportMetadata
         [JsonProperty("sepa_debit[iban]")]
         public string SepaDebitIban { get; set; }
 
+        [JsonProperty("sepa_debit[ideal]")]
+        public string SepaDebitIdealSource { get; set; }
```

Comment:
I chose to add "Source" at the end as it takes a source id and in the future we might add an `ideal` hash so wanted to protect future us